### ISSUE TITLE
Put clear link to original project and MIT licence

### ISSUE THREAD

**Issue by macournoyer** [llSourcell/Chatbot-AI#3]
I see you added a link to my GitHub profile at the bottom of the README, thx for that.

But please add a link to **the project page** at the top so everyone can find it and contribute: https://github.com/macournoyer/neuralconvo. Ideally, use the fork feature of GitHub so there's a link at the top and it's clear that this is a fork.

Also, I've released the code under MIT, which states:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

So please put back the last section of the README about the license [that you removed](https://github.com/llSourcell/Chatbot-AI/commit/86abfe8958de925c9f48cc4e1441ed6f792f6e1d).

Thanks in advance for respecting my work!

**Comment by juharris** [comment]
Thanks for pointing this out @macournoyer, I had no idea your repo existed. Do you still maintain your repo? I see that you have a word2vec branch, any plans to finish that integration?
Also maybe you can help us fix #2.

**Comment by macournoyer** [comment]
Yes still maintaining it. However, I do not have time to answer all issues. You can browse old ones for similar issues: https://github.com/macournoyer/neuralconvo/issues?q=is%3Aissue+is%3Aclosed.

Make sure to update to the latest lua rocks and clean the data/*.t7 files before re-training.

word2vec branch didn't produce good results sadly... not sure why.

**Comment by llSourcell** [comment]
Hey @macournoyer i'll do this by the end of the day i have absolutely zero intention of crediting of of your work

**Comment by macournoyer** [comment]
Any updates on this? Needs at least the license put back.
